sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -1382,7 +1387,7 @@ private Globally(String name, GlobalCombineFn<? super InputT, ?, OutputT> fn,
     public void populateDisplayData(DisplayData.Builder builder) {
       super.populateDisplayData(builder);
 
-      Combine.populateDisplayData(builder, fn, fnClass);
+      Combine.populateDisplayData(builder, fn, fnDisplayData);
```

Comment:
Can you add comments here about the choice of calling `populateDisplayData` vs. `builder.include(...)`?